packages/hono/src/index.ts

```diff
@@ -266,7 +275,14 @@ const generateHandlers = async (
 
         if (hasZValidator) {
           if (output.override.hono.validator === true) {
-            validatorImport = `\nimport { zValidator } from '${outputPath}.validator';`;
+            const validatorPath = output.override.hono.validatorOutputPath
```

Comment:
we can maybe extract it to a function since we use it 3 times?

Reply:
@anymaniax 

Sure, we can it.
But extracting it to a function wouldn't simplify the process, and I thought it would just make things harder to understand, so I didn't do it:

```diff
-  const validatorPath = output.override.hono.validatorOutputPath
-    ? getValidatorOutputRelativePath(
-        output.override.hono.validatorOutputPath,
-        handlerPath,
-      )
-    : `${outputPath}.validator`;
-
+  const validatorPath = getValidatorOutputPath(
+    output.override.hono.validatorOutputPath,
+    handlerPath,
+    outputPath,
+  )
```

Reply:
Ok I wasn’t convinced neither so let’s go like that